doc/src/cylc-user-guide/cug.tex

```diff
@@ -4658,11 +4664,11 @@ \subsubsection{Selecting Partial Parameter Ranges}
 \begin{lstlisting}
 [cylc]
     [[parameters]]
-        run = 1..10 # 01, 02, ..., 10
-        runx = 1..03  # 01, 02, 03 (note `03' to get correct padding)
+        run = 1..10 # 1, 2, ..., 10
```

Comment:
This still results in 01, 02, ... 10?

Reply:
The numbers are now stored internally as `int` 1, 2, ..., 10, but will then have a default template like `_run%(run)02d`, so the suffixes will continue to be `_run01`, `_run02`, ..., `_run10`.